<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>608</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>23</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Open">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>OpenDevice</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Close">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>CloseDevice</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Start">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>StartGrab</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Stop">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>StopGrab</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="CammerWidget" name="widget" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="label_Statistic">
       <property name="text">
        <string>Statistics</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CammerWidget</class>
   <extends>QWidget</extends>
   <header location="global">src/CammerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
